### PR TITLE
refactor(website): re-use mitmachen page

### PR DESCRIPTION
**Pull request by Itrulia**
Some getInitialProps was kept due to them loading extra data.
We need a better way to reuse those than we currently have but it's out of scope for this.

### CODE REVIEW

File: libs/utils/website/src/lib/pages/subscribe-page.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        }
+
+        return preselectedMemberPlan && additionalMemberPlans !== 'all'
+          ? [preselectedMemberPlan]
+          : parentFiltered
+      }}
+      deactivateSubscriptionId={
+        props.deactivateSubscriptionId ?? (deactivateSubscriptionId as string | undefined)
+      }
+      failureURL=""
```
Comment (michael-scheurer) [inline libs/utils/website/src/lib/pages/subscribe-page.tsx:57]: This connection got lost, but is important. Could we solve it by merging first the SHARE-42 story?

**Review by michael-scheurer — COMMENTED** [review]
That's the kind of PRs I like the most ;) Getting rid of repetitive code. Great job! Found one potential issue.

File: libs/utils/website/src/lib/pages/subscribe-page.tsx (hunk 1 above)
Comment (Itrulia) [reply libs/utils/website/src/lib/pages/subscribe-page.tsx:57]: SHARE-43? That's the point why this is empty, as SHARE-43 will be merged first.

**Comment by elias-summermatter** [comment]
@Itrulia can we merge this now?

**Comment by Itrulia** [comment]
@elias-summermatter yeah just had to resolve merge conflicts

**Pull request merged**